scripts/run_integration_tests.sh

```diff
@@ -61,7 +86,7 @@ fi
 
 declare -i timeout_secs=30
 declare -i i=0
-while (( !PID && i < timeout_secs ))
+while [[ -z "${PID}" && i -lt timeout_secs ]] 
```

Comment:
1. Why this change? Seems like just another way of doing exactly the same thing.
2. I don't think this works... if you're using normal expression evaluation (`[[ expression ]]`) instead of arithmetic evaluation (`(( _expression_ ))`), then you need to dereference variables with `$var`; otherwise you're just comparing the strings of the variable names.

Reply:
Agreed, this change was not necessary and should have been omitted from the commit: will revert.